moveit_core/robot_model/include/moveit/robot_model/robot_model.hpp

```diff
@@ -265,7 +268,13 @@ class RobotModel
    * what you went for. Instead, updateStateWithLinkAt(getRigidlyConnectedParentLinkModel(grasp_frame), ...)
    * will actually warp wrist (and all its descendants).
    */
-  static const moveit::core::LinkModel* getRigidlyConnectedParentLinkModel(const LinkModel* link);
+  static const LinkModel* getRigidlyConnectedParentLinkModel(const LinkModel* link, Eigen::Isometry3d& transform,
+                                                             const JointModelGroup* jmg = nullptr);
+  static const LinkModel* getRigidlyConnectedParentLinkModel(const LinkModel* link, const JointModelGroup* jmg = nullptr)
```

Comment:
Do we need these two functions? Wouldn't this be sufficient:

```
static const LinkModel* getRigidlyConnectedParentLinkModel(const LinkModel* link, Eigen::Isometry3d& transform = Eigen::Isometry3d(), const JointModelGroup* jmg = nullptr);
```
That way we don't break the existing API and don't add another function. Downside would be that you cannot call it with just two arguments but I think that is fine